tests/src/test/resources/com/powsybl/openrao/tests/features/epic5_max_margin_stop_criterion/US5_1.feature

```diff
@@ -15,6 +15,16 @@ Feature: US 5.1: Maximum margin stop criterion
     Then the worst margin is 500.0 MW
     Then 0 remedial actions are used in preventive
 
+  @fast @rao @mock @ac @preventive-only
+  Scenario: US 5.1.0.b: use relevant number of decimals for margin and cost logging
+    Given network file is "common/TestCase12Nodes.uct"
+    Given crac file is "epic5/SL_ep5us1b.json"
+    Given configuration file is "common/RaoParameters_posMargin_ampere.json"
+    When I launch search_tree_rao
+    Then its security status should be "SECURED"
+    Then the worst margin is -0.0001 MW
```

Comment:
I think this step has a tolerance of a few MW; are you sure it is testing what you're expecting it to ?